teams/vsa/teams/sitewide-content/how-to-do-different-tasks/how-to-update-the-style-guide.md

```diff
@@ -5,7 +5,7 @@ Sitewide Content is responsible for maintaining and updating the [VA.gov content
 2. Click on the **src** folder.
 3. Click on the **_content-style-guide** folder.
 4. Find the folder for the content style guide page you want to work on.
-5. Make any changes and save as a branch labeled as something to help identify the work being done. **Note:** Do not make changes to the master branch.
+5. Make any changes and save as a branch labeled as something to help identify the work being done. **Note:** Do not make changes to the main branch.
```

Comment:
```suggestion
5. Make any changes and save as a branch labeled as something to help identify the work being done. **Note:** Do not make changes to the master branch.
```
Since it's talking about the https://github.com/department-of-veterans-affairs/vets-design-system-documentation repo